### PR TITLE
rndecodetemplate

### PR DESCRIPTION
**Pull request by divyaswarnkar**

### CODE REVIEW

**Comment by divyaswarnkar** [comment]
@derek1ee can you please review - template for RosettaNet decode

File: templates/manifest.json [hunk 1]
```diff
@@ -40,5 +40,7 @@
     "twitter-to-outlook.json",
     "twitter-to-smtp.json",
     "veter.json",
-    "x12-to-as2.json"
+    "x12-to-as2.json",
+    "rosettanet-decode-response.json",
```
Comment (derek1ee) [inline templates/manifest.json]: Order alphabetically.

File: templates/rosettanet-decode-response.json [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    "name": "ad5addf597fc45d89143f34c9dcb39b9",
+    "properties": {
+        "author": {
+            "displayName": "Microsoft"
+        },
+        "categoryNames": [
+            "enterprise_integration"
+         ],
+        "description": "Decode RosettaNet message, send it to blob and return encoded signal to partner",
+        "displayName": "Decode RosettaNet message, send it to Blob and return encoded signal to partner",
```
Comment (derek1ee) [inline templates/rosettanet-decode-response.json]: Make sure these don't clip in the UI.

File: templates/rosettanet-decode-response.json [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+           {
+                "type": "request",
+                "displayName": "${Resources.REQUEST_TRIGGER_DISPLAYNAME}",
+                "iconUri": "${Images.RequestIcon}",
+                "brandColor": "#009DA5"
+            },
+            {
+                "type": "RosettaNetDecode",
+                "displayName": "RosettaNet Decode",
+                "iconUri": "https://latemplateartifacts.blob.core.windows.net/rosettanet/rosetta-net-connector-100.png",
```
Comment (derek1ee) [inline templates/rosettanet-decode-response.json]: Who owns this blob?

File: templates/manifest.json (hunk 1 above)
Comment (divyaswarnkar) [reply templates/manifest.json]: will do

File: templates/rosettanet-decode-response.json (hunk 2 above)
Comment (divyaswarnkar) [reply templates/rosettanet-decode-response.json]: ok

File: templates/rosettanet-decode-response.json (hunk 3 above)
Comment (divyaswarnkar) [reply templates/rosettanet-decode-response.json]: I dont know, I just copied it from an existing template. I can upload the icon in my storage if thats preferable

**Pull request merged**